models/src/controller.rs

```diff
@@ -131,10 +135,32 @@ pub fn controller_deployment(
     max_concurrent_update: String,
     update_window_start: String,
     update_window_stop: String,
+    scheduler_cron_expression: String,
 ) -> Deployment {
     let image_pull_secrets =
         image_pull_secret.map(|secret| vec![LocalObjectReference { name: Some(secret) }]);
 
+    // users can specify update time window or scheduler to schedule brupop jobs, but users aren't only allowed to use them simultaneously.
+    let mut scheduler = String::new();
```

Comment:
I think it's a "false warning" since scheduler value will be changes in the following `if` condition. But I think it's better to use expression oriented syntax like this 
```
let scheduler = if {....};
```

Reply:
If the variable is never read, e.g.

do_something(schedule), then why does it exist?